packages/sensors_plus/sensors_plus/example/integration_test/sensors_plus_test.dart

```diff
@@ -22,4 +23,68 @@ void main() {
     });
     expect(await completer.future, isNotNull);
   });
+
+  testWidgets('Can subscribe to accelerometerEvents and get expected events',
+      (WidgetTester tester) async {
+    final completer = Completer<AccelerometerEvent>();
+    StreamSubscription<AccelerometerEvent> subscription;
+    subscription = accelerometerEvents.listen((AccelerometerEvent event) {
+      completer.complete(event);
+      subscription.cancel();
+    });
+
+    final event = await completer.future;
+    expect(
+        await event.toString(),
+        AccelerometerEvent(0.0, 9.776321411132812, 0.812345027923584)
```

Comment:
I was curious if such a parameter would meet the requirements for integration testing, and I checked the research material and did not find the source of this parameter. Perhaps this is a random parameter from the simulator?

So we can do this?
@miquelbeltran

Reply:
I have no idea, that's just what the emulator returns. It would be nice to add a commentary explaining that these values are what the emulator returns at the time of the test.

Reply:
Okay, I'll spend another day determining why the simulator returns such values. I can try to understand 9.8 as the acceleration of gravity.

Reply:
I find it.

this [link](http://josejuansanchez.org/android-sensors-overview/accelerometer/README.html#:~:text=When%20the%20device%20lies%20flat%20on%20a%20table%2C%20the%20acceleration%20value%20is%20%2B9.81%2C%20which%20correspond%20to%20the%20acceleration%20of%20the%20device%20(0%20m/s%5E2)%20minus%20the%20force%20of%20gravity%20(%2D9.81%20m/s%5E2).)